### PR TITLE
OADP cannot create backups for customer-created volume snapshots

### PR DESCRIPTION
**Pull request by RichardHoch**
OADP 1.2, OCP 4.10+

Resolves https://issues.redhat.com/browse/OADP-1652 by adding a note to https://access.redhat.com/documentation/en-us/openshift_container_platform/4.13/html-single/backup_and_restore/index#backing-up-applications

Preview: 2nd comment labeled IMPORTANT in https://60877--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.html

QE approved.

### CODE REVIEW

File: backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc [hunk 1]
```diff
@@ -15,6 +15,11 @@ include::snippets/technology-preview.adoc[]
 
 If your cloud provider has a native snapshot API or supports xref:../../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc#oadp-backing-up-pvs-csi_backing-up-applications[Container Storage Interface (CSI) snapshots], the `Backup` CR backs up persistent volumes by creating snapshots. For more information, see the xref:../../../storage/container_storage_interface/persistent-storage-csi-snapshots.adoc#persistent-storage-csi-snapshots-overview_persistent-storage-csi-snapshots[Overview of CSI volume snapshots] in the {product-title} documentation.
 
+[IMPORTANT]
+====
+OADP can back up only volue snapshots that were created by OADP. OADP cannot back up volume snapshots that were created on other software.
```
Comment (kaovilai) [inline backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc:20]:
```suggestion
OADP can back up only volume snapshots that were created by OADP. OADP cannot back up volume snapshots that were created on other software.
```

**Comment by RichardHoch** [comment]
@kaovilai Made the change. Please review again.
@PrasadJoshi12 Please review this PR.

**Review by kaovilai — CHANGES_REQUESTED** [review]

**Comment by RichardHoch** [comment]
@kaovilai Changed as suggested.

**Review by PrasadJoshi12 — APPROVED** [review]

File: backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc [hunk 2]
```diff
@@ -6,7 +6,7 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-You back up applications by creating a xref:../../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc#oadp-creating-backup-cr_backing-up-applications[`Backup`] custom resource (CR).
+You back up applications by creating a xref:../../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc#oadp-creating-backup-cr_backing-up-applications[`Backup`] custom resource (CR) using OADP.
```
Comment (dfitzmau) [inline backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc:9]: For accessibility reasons, the link text does not equate to the target link heading (Backup > Creating a Backup CR). Using inline link text can become difficult for this reasons. Also, the text renders in monospace instead of the hyperlink formatting. 

Suggestion:

> You back up applications by creating a `Backup` custom resource (CR) by using OADP. See xref[Backing up and restoring applications].

Additionally, "using" should be "by using"

Screenshot from 2023-06-08 10-00-26

File: backup_and_restore/application_backup_and_restore/backing_up_and_restoring/restoring-applications.adoc [hunk 3]
```diff
@@ -6,7 +6,7 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-You restore application backups by creating a xref:../../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/restoring-applications.adoc#oadp-creating-restore-cr_restoring-applications[`Restore` custom resources (CRs)].
+You restore application backups by creating a xref:../../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/restoring-applications.adoc#oadp-creating-restore-cr_restoring-applications[`Restore` custom resources (CRs)] using OADP.
```
Comment (dfitzmau) [inline backup_and_restore/application_backup_and_restore/backing_up_and_restoring/restoring-applications.adoc:9]: Same suggestion as my initial comment. I think all the other links should follow the same pattern as the link text does not natch the target heading and the formatting of the text does not indicate a hyperlink.

File: backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc [hunk 4]
```diff
@@ -15,6 +15,11 @@ include::snippets/technology-preview.adoc[]
 
 If your cloud provider has a native snapshot API or supports xref:../../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc#oadp-backing-up-pvs-csi_backing-up-applications[Container Storage Interface (CSI) snapshots], the `Backup` CR backs up persistent volumes by creating snapshots. For more information, see the xref:../../../storage/container_storage_interface/persistent-storage-csi-snapshots.adoc#persistent-storage-csi-snapshots-overview_persistent-storage-csi-snapshots[Overview of CSI volume snapshots] in the {product-title} documentation.
 
+[IMPORTANT]
+====
```
Comment (dfitzmau) [inline backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc:19]: OADP is not spelled out in its full term in this module. Should it be spelled out fully on its first occurrence?

File: backup_and_restore/index.adoc [hunk 5]
```diff
@@ -74,12 +74,12 @@ If you do not want to back up PVs by using snapshots, you can use link:https://r
...
 
-You restore applications by creating a xref:../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/restoring-applications.adoc#oadp-creating-restore-cr_restoring-applications[`Restore`] CR. You can configure xref:../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/restoring-applications.adoc#oadp-creating-restore-hooks_restoring-applications[restore hooks] to run commands in init containers or in the application container during the restore operation.
+You restore applications by creating a xref:../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/restoring-applications.adoc#oadp-creating-restore-cr_restoring-applications[`Restore`] CR using OADP. You can configure xref:../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/restoring-applications.adoc#oadp-creating-restore-hooks_restoring-applications[restore hooks] to run commands in init containers or in the application container during the restore operation.
```
Comment (dfitzmau) [inline backup_and_restore/index.adoc:83]: Should restore be highlighted in back ticks?

I'm unsure if `init` should receive the same formatting because I see some instances in the doc set where it does receive this formatting and others where it does not. For example, see (https://access.redhat.com/documentation/en-us/openshift_container_platform/4.13/html-single/backup_and_restore/index#oadp-creating-restore-hooks_restoring-applications)[Creating restore hooks]

Comment (dfitzmau) [reply backup_and_restore/index.adoc:83]: Would front-loading "during the restore operation" be better, because this important limitation is only realized at the end of the sentence.

**Comment by dfitzmau** [comment]
Hi @RichardHoch . Nice work on the PR. I added comments inline, but I'll need @mburke5678 to do a second scan of my review comments.

**Review by kaovilai — APPROVED** [review]

File: backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc (hunk 4 above)
Comment (mburke5678) [reply backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc:19]: Spell out acronyms on the first occurrence in each module and assembly.

File: backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc (hunk 2 above)
Comment (RichardHoch) [reply backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc:9]: Fixed.

File: backup_and_restore/index.adoc (hunk 5 above)
Comment (RichardHoch) [reply backup_and_restore/index.adoc:83]: @dfitzmau @mburke5678 The original sentence is hard to parse (I parsed it differently than you did, @dfitzmau). I've replaced it with a slightly modified version of the first sentence in the cross-referenced section. The text is now easier to read and leaves the details to the section itself. WDYT?

File: backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc [hunk 6]
```diff
@@ -6,20 +6,30 @@ include::_attributes/common-attributes.adoc[]
...
 
 :FeatureName: The `CloudStorage` API for S3 storage
 include::snippets/technology-preview.adoc[]
 
-If your cloud provider has a native snapshot API or supports xref:../../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc#oadp-backing-up-pvs-csi_backing-up-applications[Container Storage Interface (CSI) snapshots], the `Backup` CR backs up persistent volumes by creating snapshots. For more information, see the xref:../../../storage/container_storage_interface/persistent-storage-csi-snapshots.adoc#persistent-storage-csi-snapshots-overview_persistent-storage-csi-snapshots[Overview of CSI volume snapshots] in the {product-title} documentation.
+* If your cloud provider has a native snapshot API or supports CSI snapshots], the `Backup` CR backs up persistent volumes (PVs) by creating snapshots. For more information about working with CSI snapshots, see xref:../../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc#oadp-backing-up-pvs-csi_backing-up-applications[Backing up persistent volumes with CSI snapshots]
```
Comment (anarnold97) [inline backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc:20]: Should this end with a period

File: backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc [hunk 7]
```diff
@@ -6,20 +6,30 @@ include::_attributes/common-attributes.adoc[]
...
 
-The `Backup` CR creates backup files for Kubernetes resources and internal images, on S3 object storage, and snapshots for persistent volumes (PVs), if the cloud provider uses a native snapshot API or the xref:../../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc#oadp-backing-up-pvs-csi_backing-up-applications[Container Storage Interface (CSI)] to create snapshots, such as {rh-storage} 4. For more information, see xref:../../../storage/container_storage_interface/persistent-storage-csi-snapshots.adoc#persistent-storage-csi-snapshots[CSI volume snapshots].
+oadp-creating-backup-cr_backing-up-applications
+
+The `Backup` CR creates backup files for Kubernetes resources and internal images, on S3 object storage, and snapshots for persistent volumes (PVs), if the cloud provider uses a native snapshot API or the Container Storage Interface (CSI) to create snapshots, such as {rh-storage} 4.
+
+For more information about CSI volume snapshots see xref:../../../storage/container_storage_interface/persistent-storage-csi-snapshots.adoc#persistent-storage-csi-snapshots[CSI volume snapshots].
```
Comment (anarnold97) [inline backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc:15]: Possibly - For more information about CSI volume snapshots**,** see xref...

File: backup_and_restore/index.adoc [hunk 8]
```diff
@@ -74,12 +74,12 @@ If you do not want to back up PVs by using snapshots, you can use link:https://r
 [id="backing-up-and-restoring-applications"]
 === Backing up and restoring applications
 
-You back up applications by creating a xref:../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc#oadp-creating-backup-cr_backing-up-applications[`Backup`] custom resource (CR). You can configure the following backup options:
+You back up applications by creating a `Backup` custom resource (CR) by using OADP. See xref:../../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc#oadp-creating-backup-cr_backing-up-applications[Creating a Backup CR].You can configure the following backup options:
```
Comment (anarnold97) [inline backup_and_restore/index.adoc:77]: I think this might be the build error.

Note the xref:
**xref:../**backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc#oadp-creating-backup-cr_backing-up-applications
Compared to 
**xref:../../../**backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc#oadp-creating-backup-cr_backing-up-applications

File: backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc (hunk 7 above)
Comment (RichardHoch) [reply backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc:13]: Fixed.

**Comment by RichardHoch** [comment]
@dfitzmau @mburke5678  Thanks for making the contents of this PR better and for helping me make it more accessible.

File: backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc [hunk 9]
```diff
@@ -6,20 +6,27 @@ include::_attributes/common-attributes.adoc[]
...
 
 :FeatureName: The `CloudStorage` API for S3 storage
 include::snippets/technology-preview.adoc[]
 
-If your cloud provider has a native snapshot API or supports xref:../../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc#oadp-backing-up-pvs-csi_backing-up-applications[Container Storage Interface (CSI) snapshots], the `Backup` CR backs up persistent volumes by creating snapshots. For more information, see the xref:../../../storage/container_storage_interface/persistent-storage-csi-snapshots.adoc#persistent-storage-csi-snapshots-overview_persistent-storage-csi-snapshots[Overview of CSI volume snapshots] in the {product-title} documentation.
+* If your cloud provider has a native snapshot API or supports CSI snapshots], the `Backup` CR backs up persistent volumes (PVs) by creating snapshots. For more information about working with CSI snapshots, see xref:../../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc#oadp-backing-up-pvs-csi_backing-up-applications[Backing up persistent volumes with CSI snapshots].
```
Comment (mburke5678) [inline backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc:18]: s/snapshots]/snapshots

File: backup_and_restore/application_backup_and_restore/backing_up_and_restoring/restoring-applications.adoc [hunk 10]
```diff
@@ -6,9 +6,9 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-You restore application backups by creating a xref:../../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/restoring-applications.adoc#oadp-creating-restore-cr_restoring-applications[`Restore` custom resources (CRs)].
+You restore application backups by creating a `Restore` custom resource (CR) by using the OpenShift API for Data Protection (OADP). See xref:../../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/restoring-applications.adoc#oadp-creating-restore-cr_restoring-applications[Creating a Restore CR].
```
Comment (mburke5678) [inline backup_and_restore/application_backup_and_restore/backing_up_and_restoring/restoring-applications.adoc:9]: Do you really create a CR by using the API? I'm not sure of the right wording. 

`creating a `Restore` custom resource (CR) that uses...`??

File: backup_and_restore/index.adoc [hunk 11]
```diff
@@ -74,12 +74,12 @@ If you do not want to back up PVs by using snapshots, you can use link:https://r
 [id="backing-up-and-restoring-applications"]
 === Backing up and restoring applications
 
-You back up applications by creating a xref:../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc#oadp-creating-backup-cr_backing-up-applications[`Backup`] custom resource (CR). You can configure the following backup options:
+You back up applications by creating a `Backup` custom resource (CR) by using OADP. See xref:../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc#oadp-creating-backup-cr_backing-up-applications[Creating a Backup CR].You can configure the following backup options:
```
Comment (mburke5678) [inline backup_and_restore/index.adoc:77]: Same here with _by using OADP_

File: backup_and_restore/index.adoc [hunk 12]
```diff
@@ -74,12 +74,12 @@ If you do not want to back up PVs by using snapshots, you can use link:https://r
...
 
-You restore applications by creating a xref:../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/restoring-applications.adoc#oadp-creating-restore-cr_restoring-applications[`Restore`] CR. You can configure xref:../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/restoring-applications.adoc#oadp-creating-restore-hooks_restoring-applications[restore hooks] to run commands in init containers or in the application container during the restore operation.
+You restore application backups by creating a `Restore` (CR) by using OADP. See xref:../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/restoring-applications.adoc#oadp-creating-restore-cr_restoring-applications[Creating a Restore CR].  You can configure xref:../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/restoring-applications.adoc#oadp-creating-restore-hooks_restoring-applications[restore hooks] to run commands in init containers or in the application container during the restore operation.
```
Comment (mburke5678) [inline backup_and_restore/index.adoc:83]: Same here with _by using OADP_

**Comment by mburke5678** [comment]
@RichardHoch One typo and a few nits. Otherwise LGTM

**Comment by RichardHoch** [comment]
@mburke5678 Thanks for your comments. GA has been delayed until Wednesday the 14h but if you could approve the PR today, i'd appreciate it.

**Comment by mburke5678** [comment]
@RichardHoch 
Did you address Darragh's [concern here regarding spelling out OADP](https://github.com/openshift/openshift-docs/pull/60877/files#r1222692997)?

**Comment by jeana-redhat** [comment]
> @RichardHoch Did you address Darragh's [concern here regarding spelling out OADP](https://github.com/openshift/openshift-docs/pull/60877/files#r1222692997)?

Removing the merge labels to make sure everyone is on the same page with the review status. Nothing jumped out to me in merge review though so I think as long as everyone is in accord this is ready :)

**Comment by RichardHoch** [comment]
@mburke5678 After I removed the phrase "by using OADP," I was left with the odd situation that the first use of OADP in backing-up-applications is in the admonition ("IMPORTANT"). I went ahead and spelled it out. Let me know what you think of that.
